app/src/main/res/values-en/strings.xml

```diff
@@ -1268,15 +1268,15 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_smoothness_description_very_bad_paved">Deep potholes, ruts or other dangerous damage</string>
     <string name="quest_smoothness_description_very_bad_sett">Big gaps, displaced, missing or irregularly shaped stones or dangerous damage</string>
     <string name="quest_smoothness_description_very_bad_paving_stones">Missing and broken stones, gaps or other dangerous damage</string>
-    <string name="quest_smoothness_description_very_bad_compacted_gravel">Some large stones, deep potholes, bumps, ruts, erosion or other dangerous damage</string>
+    <string name="quest_smoothness_description_very_bad_compacted_gravel">Some large stones, deep potholes, bumps, ruts, erosion or other hazardous damage</string>
     <string name="quest_smoothness_description_horrible">Properly usable only by off-road vehicles or on foot</string>
     <string name="quest_smoothness_description_very_horrible">Properly usable only by specialized off-road vehicles or on foot</string>
     <string name="quest_smoothness_description_impassable">Unfit for any vehicle, only passable on foot</string>
     <string name="quest_smoothness_wrong_surface">Surface is different</string>
     <string name="quest_smoothness_surface_value">The surface was tagged as:</string>
     <string name="quest_smoothness_obstacle">There is an obstacle</string>
-    <string name="quest_smoothness_obstacle_hint">Please provide a general answer for the smoothness. Single obstacles like kerbs, speed bumps or cattle grids can be ignored. Repeated or unexpected obstacles should count towards the answer.</string>
-    <string name="quest_smoothness_hint">🚲, 🚗, 🚙, … indicate for which vehicles the surface should be okay to use. What matters is the big picture rather than single damages.</string>
+    <string name="quest_smoothness_obstacle_hint">Please provide an answer for the overall smoothness. Single obstacles like kerbs, speed bumps or cattle grids can be ignored. Repeated or unexpected obstacles should count towards the answer.</string>
```

Comment:
Do you want to use your term for cattle grid here too?
```suggestion
    <string name="quest_smoothness_obstacle_hint">Please provide an answer for the overall smoothness. Single obstacles like kerbs, speed bumps or cattle grids can be ignored. Repeated or unexpected obstacles should count towards the answer.</string>
```

Reply:
Too long.

Reply:
My change or @ZeLonewolf 's? I can't find it now, but I thought it was only a few characters different from "cattle grids". The whole current thing is only a few characters longer than it was.

Reply:
The term is "cattle guard".  There was another string when it was a much longer phrase those, but I restored it to the original text.